.github/workflows/quay.yml

```
@@ -0,0 +1,19 @@
+name: quay
+
+on: push
+
+# on:
+#   push:
+#     branches:
+#       - master
+
+jobs:
+  push:
+    runs-on: ubuntu-latest
+    env:
+      header: "Content-Type:application/json"
+      payload: "{\"commit\":\"$GITHUB_SHA\",\"ref\":\"$GITHUB_REF\",\"default_branch\":\"master\"}"
+      url: "${{ secrets.QUAY_WEBHOOK_URL }}"
+    steps:
+    - name: update container on quay.io
+      run: curl -H $header -d $payload $url
```

I think we should still deflate the header environment variable and just hardcode the request header here.

---

Also, I'm almost positive you can get the payload variable working by single-quoting it:

```suggestion
      run: curl -H $header -d '$payload' $url
```

---

Nope, wasn't the problem